NewForce-Cohort-9/newforce-c9-tabloid-full-stack-TabloidFullStack
# Daniel branch2

Description:

I was able to create views for Tags and all four tags should populate the broswer. In VSCode you should see modules labeled as such: Tags.js, TagDetails.js, TagPageHeader.js, Tags,css. There are two separate modules that are very similar: UpdateTagForm.js and CreateTagForm.js. These tags are for the links and buttons that will allow us to create and edit tags but they are currently nonfunctional. All that should happen is you should see the tags in the browsers if you navigate to localhost3000/tags.

Testing Instructions

git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout
If you don't see the most recent version of my files, git pull origin
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-
Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-
9.Click the execute with debugger solid green button to launch the app
In browser, please go to Login on the nav bar and log in with this email: [admin@example.com](mailto:admin@example.com)
This should redirect you back to the home page.
Log out should be showing on the nav bar now. Please click this to log out.
You should see a friendly message letting you know you are logged out of the application.
CheckList:

My code follows the style guidelines for this project.
My code is easy to follow and read.
The tags display in the browser and you can see them and the code works.
my changes generate no warnings or errors.
I have added test instructions that prove my fis is effective or that my feature works.
